.ci/env/apt.sh

```diff
@@ -62,6 +62,34 @@ function install_qemu_emulation {
     sudo apt-get install -y qemu-user-static
 }
 
+function install_llvm_version {
+    sudo apt-get install -y curl
+    curl -o llvm.sh https://apt.llvm.org/llvm.sh
+    chmod u+x llvm.sh
+    sudo ./llvm.sh "$1"
+    sudo update-alternatives --install /usr/bin/clang clang "/usr/bin/clang-$1" "$1"
+    sudo update-alternatives --install /usr/bin/clang++ clang++ "/usr/bin/clang++-$1" "$1"
+}
+
+function build_sysroot {
+    # Usage:
+    #   build_sysroot working-dir arch os-name out-dir
+    # where the architecture and OS name need to be recognised by debootstrap,
+    # e.g. arch=arm64, os-name=jammy. The output directory path is relative to
+    # the working directory
+    mkdir -p "$1"
+    pushd "$1" || exit
+    sudo apt-get install -y debootstrap build-essential
+    sudo debootstrap --arch="$2" --verbose --include=fakeroot,symlinks,libatomic1 --resolve-deps --variant=minbase --components=main,universe "$3" "$4"
+    sudo chroot "$4" symlinks -cr .
+    sudo chown "${USER}" -R "$4"
+    rm -rf "${4:?}"/{dev,proc,run,sys,var}
```

Comment:
Can you please explain how and for what these rm functions will work? Can it affect user?

Reply:
This is just to save on a few bytes when caching the sysroot. We only need the sysroot for includes and libraries, and not for running a chroot, so we rm some of the executables. The use of "${4:?}" will ensure that we don't remove /dev /proc, etc. The list might not be exhaustive here, but it is at least a start for what can be trimmed away.

Reply:
I see, thanks! I relaunched a private ci job, previous one was pretty green, but it's just for doublechecking. When it will be finished, I'll approve this pr.